src/main/scala/org/globalforestwatch/summarystats/afi/AFiDF.scala

```diff
@@ -40,5 +40,7 @@ object AFiDF extends SummaryDF {
       }
       .toDF("id", "error", "dataGroup", "data")
       .select($"id.*", $"error.*", $"dataGroup.*", $"data.*")
+      .filter($"location_id" =!= -2)
```

Comment:
Sorry, should've been clearer: we should filter these out at the very beginning, like if AFiCommand class, before we even run the analysis.